src/modules/current.js

```diff
@@ -20,6 +20,7 @@ export const getDefaultFromUi = (state, action) => {
     return {
         ...state,
         [BASE_FIELD_TYPE]: ui.type,
+        outputType: ui.input.type,
```

Comment:
This requires knowledge about exact variable names in the redux store outside of the reducer file. Could we use a selector?

Reply:
I was wondering exactly the same and checked other parts of the code, no selectors are used.
But we do have a selector for `ui.input.type` and probably all the other too.

Reply:
Right, I really think we should use selectors all over.

Reply:
It's not possible to use selectors because they need the whole `state` of the store.
Here only the `current` section of the state is available.